website/templates/log_list.mako

```diff
@@ -46,12 +46,17 @@
                                 <span class="contributor-anonymous">A user</span>
                             </span>
                                 <span data-bind="ifnot:log.anonymous">
+                                    <span data-bind="if: log.user.registered">
                                     <span data-bind="if: log.userURL">
                                         <a class="overflow" data-bind="text: log.userFullName || log.apiKey, attr: {href: log.userURL}"></a>
                                     </span>
                                     <span data-bind="ifnot: log.userURL">
                                         <span class="overflow" data-bind="text: log.userFullName"></span>
                                     </span>
+                                    </span>
+                                    <span data-bind="ifnot: log.user.registered">
+                                        Deactivated User
```

Comment:
The two logs are using the two different APIs. `logTextParser.js` is the newer one using API v2 and will replace the one in the Project dashboard *soon.